spec/compiler/parser/parser_spec.cr

```diff
@@ -965,6 +965,9 @@ module Crystal
     it_parses %(macro foo\n"\\'"\nend), Macro.new("foo", body: Expressions.from([%("\\'"\n).macro_literal] of ASTNode))
     it_parses %(macro foo\n"\\\\"\nend), Macro.new("foo", body: Expressions.from([%("\\\\"\n).macro_literal] of ASTNode))
 
+    it_parses "macro foo;bar(end: 1);end", Macro.new("foo", body: Expressions.from(["bar(".macro_literal, "end: 1);".macro_literal] of ASTNode))
+    it_parses "def foo;bar(end: 1);end", Def.new("foo", body: Expressions.from([Call.new(nil, "bar", named_args: [NamedArgument.new("end", 1.int32)])] of ASTNode))
```

Comment:
```suggestion
    it_parses "macro foo; bar(end: 1); end", Macro.new("foo", body: Expressions.from(["bar(".macro_literal, "end: 1);".macro_literal] of ASTNode))
    it_parses "def foo; bar(end: 1); end", Def.new("foo", body: Expressions.from([Call.new(nil, "bar", named_args: [NamedArgument.new("end", 1.int32)])] of ASTNode))
```

Reply:
What difference does it make?

Reply:
Readability, I believe we ought to stick to self-imposed standards (that's why we have formatter) not only in the code but also in specs.

Reply:
Please take a look at the macro specs immediately preceding these ones. They omit unnecessary white space following expression delimiters. In macro literals, white space *is* relevant. Changing the example code requires changing the expected values, which now would include leading and trailing white space in the macro literals.

Reply:
Fair enough, it's not worth the hassle.